doc/toxic.conf.5.asc

```diff
@@ -76,7 +76,10 @@ OPTIONS
         Enable friend connection change notifications. true or false
 
     *nodelist_update_freq*;;
-        How often in days to update the DHT nodes list. (0 to disable updates)
+        How often in days to update the DHT nodes list. (integer; 0 to disable)
+
+    *autoupdate_freq*;;
```

Comment:
This says autoupdate, but below it says autosave.

Reply:
Oops, fixed that locally but didn't commit it.